package.json

```diff
@@ -12,7 +12,8 @@
     "clean": "rimraf lib es dist",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "jest --config=jest.json",
-    "test-ci": "yarn flow:check && yarn eslint && yarn test && yarn translations:check",
+    "test-ci": "yarn flow:check && yarn eslint && yarn test && yarn translations:check && cypress run",
```

Comment:
I'm not sure this will work unless you `yarn start` it.

Actually I believe we should do integration tests on CI against the build+deployed version, not the localhost running one.

Reply:
I suggest to `yarn start` it for now and think about testing against the deployed version on another issue :slightly_smiling_face:

Reply:
I agree

Reply:
I added `yarn start` into `test-ci` script, but this fails because are missing one dependency.

![screen shot 2018-05-28 at 12 36 07](https://user-images.githubusercontent.com/17720925/40610761-ec41f1c0-6273-11e8-80df-7304c2a0f159.png)

What do we do?